.github/workflows/sub-ci-unit-tests-docker.yml

```diff
@@ -53,10 +53,15 @@ jobs:
       - name: Run all tests
         run: |
           docker pull ${{ vars.GAR_BASE }}/${{ vars.CI_IMAGE_NAME }}@${{ inputs.image_digest }}
-          docker run -t \
+          docker run --tty \
           -e RUN_ALL_TESTS=1 \
           -e FEATURES="journald prometheus filter-reload" \
           -e NETWORK="${{ inputs.network || vars.ZCASH_NETWORK }}" \
+          -e RUST_LOG=${{ env.RUST_LOG }} \
+          -e RUST_BACKTRACE=${{ env.RUST_BACKTRACE }} \
+          -e RUST_LIB_BACKTRACE=${{ env.RUST_LIB_BACKTRACE }} \
+          -e COLORBT_SHOW_HIDDEN=${{ env.COLORBT_SHOW_HIDDEN }} \
```

Comment:
Are these actually set anywhere?

Reply:
Here 
<img width="918" alt="image" src="https://github.com/user-attachments/assets/b71da07d-864e-4700-8d6f-87dac9995fe8" />

Reply:
I'm now thinking we could move these to the yml files so we could set them individually for each workflow or job. For example, setting `RUST_LOG` to warn would significantly reduce the massive logs we have for all workflows. And if there was something we needed to debug, and the logs would be insufficient, we could increase the log level for that individual failing job. Scrolling the logs takes quite a lot of time currently.